container-images/mds-agency/webpack.config.ts

```diff
@@ -1,3 +1,3 @@
 import webpack from '@mds-core/mds-webpack-config'
 
-export default webpack.Bundle('../../packages/mds-agency/server.ts').UsingDefaultConfig()
+export default webpack.Bundle('../../packages/mds-agency/server').UsingDefaultConfig()
```

Comment:
I know the linter is smart enough to figure this type of thing out in the imports, any chance we could get some rules to apply to this scenario as well?

Reply:
This was only changed to allow loading the entry point from either `server.ts` or `server/index.ts`